environments/sbox/backend_lb_config.yaml

```diff
@@ -93,4 +93,5 @@ gateways:
         component: docmosis
         cookie_based_affinity: Enabled
         ssl_enabled: true
-        host_name_prefix: docmosis
\ No newline at end of file
+        host_name_prefix: docmosis
+        http_to_https_redirect: true
```

Comment:
is this just for like BA's accessing the web ui?

Reply:
Yeah, just for UI - after login it goes to http and a 404

Reply:
that's masking an issue then, it shouldn't be sending you to http, either x-forwarded-* headers are missing (unlikely but check), or something needs to be done at docmosis to tell it that its behind a proxy

Reply:
tried few things and couldn't find anything specific and makes it difficult as we are on a old version of Docmosis.